### DISCUSSION TITLE
You forgot to create a new RC tag Markus ;)

### DISCUSSION THREAD

**Discussion by matsn0w** [rebing/graphql-laravel#842] [General]
@mfn

A couple of PR's have been merged without a new RC tag... Kind reminder :)

**Comment by mfn** [comment]
Thanks for the reminder, it's expected as the main page reflects master branch but we're still on pre-release RC levels. I do plan a release "soon" but no specific ETA!

**Comment by mfn** [comment]
PS: [RC5 was tagged](https://github.com/rebing/graphql-laravel/discussions/848)
